pkgs/tools/games/minecraft/hmcl/default.nix

```
@@ -0,0 +1,62 @@
+{ lib, stdenv
```

Looks a bit inconsistent
```suggestion
{ lib
, stdenv
```